scripts/camViewer

```diff
@@ -399,17 +414,35 @@ fi
 IS_ACQUIRING=$(caget -t -n "$CAMPVFULL":Acquire_RBV)
 if isNum "$IS_ACQUIRING"; then
     if [ "$IS_ACQUIRING" -eq 0 ]; then
-	echo 'start acquiring for camera ' "$CAMNAME"
-	caput -n "$CAMPVFULL":Acquire 1
+        echo 'start acquiring for camera ' "$CAMNAME"
+        caput -n "$CAMPVFULL":Acquire 1
     fi
 else
     echo 'cannot check Acquiring PV, try to open camera viewer anyways....'
 fi
 
+if [[ -z "$IS_REMOTE_SSH" ]]; then
+    if [ "$hutch" == "kfe" ]; then
+        echo "Not in SSH, connecting to kfe-console..." >&2
+        ssh -t kfe-console "IS_REMOTE_SSH=1 $EXE --camerapv '$CAMPVFULL' --instrument '$hutch' --pvlist '$PVLIST' --rate '$RATE' ; bash"
```

Comment:
One small detail, `rate` defaults to `1` inside camviewer if it is left as `-1` in this script. I think that's why there's a rate switch below in the non-ssh variant. I'm not sure how much this matters.

References:

- https://github.com/pcdshub/engineering_tools/blob/ffc6675a8e19a14dc0b7c01e3d884c51f740c9b4/scripts/camViewer#L194
- https://github.com/pcdshub/camviewer/blob/dafba34800b613c778124811f0712a2010190f7e/camviewer_ui_impl.py#L541

Reply:
Possibly it would make sense to default rate to 5 in this script instead of -1 and then we no longer need to keep track of multiple variants